sendgrid/helpers/permissions/unit_tests.py

```diff
@@ -0,0 +1,437 @@
+from permissions import Permissions
+
+'''
+    Testing features from first bullet point:
+    Allow you to add create, delete, read, update permissions for each of the groups defined here -- these can be hard coded in
+'''
+
+
+# test case 1:
+# add new permissions list "test" with permissions "test.test" and "test.test1"
+def test1():
```

Comment:
Unlike regular methods, unit test method names generally tell you what they do (and are [super long](https://github.com/sendgrid/sendgrid-python/blob/4ae3c96a7772433918c489c28b8f79206e276ed2/test/test_mail.py#L413)  as a result). These method names should be descriptive.  For instance,
```
def test_add_permissions_list():
```